python/utilities.py

```diff
@@ -372,14 +366,18 @@ def ewma(current_val, past_ewma, weight, is_angle):
         is_angle (bool): True when getting the field is an angle, false otherwise.
 
     Returns:
-        float: The exponentially weighted moving average.
+        float: The exponentially weighted moving average, bounded in [0, 360) if is_angle is true.
     """
     if past_ewma is None:
         return current_val
 
     if is_angle:
         _, current_val = mitsutaVals((past_ewma, current_val))
```

Comment:
Just to make sure, `ewma()` doesn't need to be able to handle angles that are bounded really weirdly right? With weirdly bounded angles, I think there's a potential for delta in `mitsutaVals()` to be outside of (-540, 540) which could yield faulty results.

If it doesn't need to handle weird angles, would it be a good idea to add the angle bounding requirements on current_val and/or past_ewma in the description for the `ewma()` function?

Reply:
Nice catch, I will add that to the docstring. These arguments are implicitly bounded in [0, 360) in the current implementation so no code needs to change:
* `current_val` comes from the second element in the tuple returned by `utils.vectorAverage()`
* `past_ewma` comes from the return value of `utils.ewma()`